terraform/ssh.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-ssh ubuntu@`terraform output ip`
+ssh ubuntu@`terraform output host_ip`
```

Comment:
Hmm. It's definitely been useful for me while developing and debugging stuff. I see your point though, that it will only work for whoever ran the provisioner last. Could we just document that limitation for now, seeing as we hope to find a solution to that problem anyway?